numpy/core/include/numpy/npy_2_complexcompat.h

```diff
@@ -0,0 +1,22 @@
+#ifndef NUMPY_CORE_INCLUDE_NUMPY_NPY_2_COMPLEXCOMPAT_H_
+#define NUMPY_CORE_INCLUDE_NUMPY_NPY_2_COMPLEXCOMPAT_H_
+
+#include <numpy/npy_math.h>
+
+#if NPY_FEATURE_VERSION < NPY_2_0_API_VERSION
+#define NPY_CSETREALF(c, r) (c)->real = (r)
+#define NPY_CSETIMAGF(c, i) (c)->imag = (r)
+#define NPY_CSETREAL(c, r)  (c)->real = (r)
+#define NPY_CSETIMAG(c, i)  (c)->imag = (r)
+#define NPY_CSETREALL(c, r) (c)->real = (r)
+#define NPY_CSETIMAGL(c, i) (c)->imag = (r)
+#else
+#define NPY_CSETREALF(c, r) npy_csetrealf(c, r)
+#define NPY_CSETIMAGF(c, i) npy_csetimagf(c, i)
+#define NPY_CSETREAL(c, r)  npy_csetreal(c, r)
+#define NPY_CSETIMAG(c, i)  npy_csetimag(c, i)
+#define NPY_CSETREALL(c, r) npy_csetreall(c, r)
+#define NPY_CSETIMAGL(c, i) npy_csetimagl(c, i)
+#endif
+
+#endif
```

Comment:
How would this get used, for instance in scipy (which wants to build with both older NumPy that does not have this header and newer NumPy that does have it)?

Reply:
I added this so that downstream packages can copy-paste it into their sources. That's what I understood from yesterday's comments. Is this correct or should I do something else?

Reply:
I guess we can rename the file later if we want to add more to it.  I wonder if there is a better place, maybe even in the docs.  But I can't quite think of one.
(We should try to eventually remember putting a link to it at least once in the release notes.)